source/compatibility.txt

```diff
@@ -0,0 +1,62 @@
+.. _php-compatibility:
+
+=============
+Compatibility
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: backwards compatibility, versions, upgrade
+
+MongoDB Compatibility
+---------------------
+
+The following compatibility table specifies the recommended version or versions
+of the {+php-library+} and extension that you can use with a specific version of MongoDB.
+
+The first column lists the version of the library and extension.
+
+.. sharedinclude:: dbx/lifecycle-schedule-callout.rst
+
```

Comment:
S: Did you remove the compatibility table legend? I think we should add it back here ([reference](https://www.mongodb.com/docs/drivers/go/current/compatibility/#compatibility-table-legend))